bin/bintray_garbage_collector.rb

```diff
@@ -0,0 +1,25 @@
+require 'dotenv'
+require './lib/garbage_collector.rb'
+
+# We might want to extra this in a file to make it easier to edit
+projects = [
+  { bintray: 'wordpress-mobile/maven/utils', github: 'wordpress-mobile/WordPress-Utils-Android' }
+]
```

Comment:
I'll do the extraction in a file when it'll be time to add another package to the list.